index.js

```diff
@@ -123,26 +126,28 @@ ExpressOAuthServer.prototype.token = function(options) {
 
 var handleResponse = function(req, res, response) {
   res.set(response.headers);
-
-  res.send(response.body);
-  res.status(response.status);
+  res.status(response.status).send(response.body);
 };
 
 /**
  * Handle error.
  */
 
-var handleError = function(e, req, res, response) {
-  if (response) {
-    res.set(response.headers);
-  }
+var handleError = function(e, req, res, response, next) {
 
-  if (e instanceof UnauthorizedRequestError) {
-    return res.status(e.code);
-  }
+  if (this.useErrorHandler === true) {
```

Comment:
This is not enough. It's nested function, so there will not be `this === ExpressOAuthServer` value until you bind `this` to the nested function. 
I recommend to use `strict mode`, so you will see this problem clearly.